skimage/io/tests/test_pil.py

```diff
@@ -97,6 +97,17 @@ def test_imread_truncated_jpg():
                   os.path.join(data_dir, 'truncated.jpg'))
 
 
+def test_jpg_quality_arg():
+    chessboard = np.load(os.path.join(data_dir, 'chessboard_GRAY_U8.npy'))
+    with NamedTemporaryFile(suffix='.jpg') as jpg:
```

Comment:
@jni, you mean what the stdlib should have done in the first place?  Sounds good to me :smile: